openshift_images/cnf-building-and-deploying-a-dpdk-payload.adoc

```diff
@@ -8,31 +9,33 @@ The Data Plane Development Kit (DPDK) base image is a base image for DPDK
 applications. It uses the Source-to-Image (S2I) build tool to automate the
 building of application images.
 
-Source-to-Image (S2I) is a tool for building reproducible, Docker-formatted
+Source-to-Image (S2I) is a tool for building reproducible and formatted
 container images. It produces ready-to-run images by injecting application
 source into a container image and assembling a new image. The new image
 incorporates the base image (the builder) and built source. For more
 information, see
 xref:../builds/build-strategies.adoc#build-strategy-s2i_build-strategies[Source-to-Image
 (S2I) build].
 
-The DPDK base image that comes preinstalled with DPDK, and a build tool used to
-create a target image with DPDK and a user provided application.
+The DPDK base image comes preinstalled with DPDK, and with a build tool that can be used to
+create a target image containing the DPDK libraries and the application provided by the user.
 
 .Prerequisites
 
 Before using the S2I tool, ensure that you have the following components installed and configured:
 
-* xref:../registry/configuring-registry-operator.adoc#configuring-registry-operator[Image Registry Operator].
+* OCP Image Registry Operator
+(See  https://docs.openshift.com/container-platform/4.4/registry/configuring-registry-operator.html[Image Registry Operator in OpenShift Container Platform].)
 
-* xref:../networking/hardware_networks/installing-sriov-operator.adoc#installing-sriov-operator[Installing the SR-IOV Network Operator].
+* SR-IOV Operator
+(See https://docs.openshift.com/container-platform/4.4/networking/hardware_networks/about-sriov.html[About SR-IOV hardware on OpenShift Container Platform].)
 
-An example DPDK-based application base on the dpdk-based image is available in
-the link:https://github.com/openshift-kni/cnf-features-deploy/tree/master/tools/s2i-dpdk/test/test-app[cnf-features-deploy] repository.
+* Performance AddOn Operator
+(See https://docs.openshift.com/container-platform-ocp/4.4/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html[About CNF Performance addon operator].)
```

Comment:
xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#performance-addon-operator-for-low-latency-nodes[[About CNF Performance Addon Operator].)

Reply:
Changed